### DISCUSSION TITLE
Order #include statement  // <Lcd Menu.h> not in last position

### DISCUSSION THREAD

**Discussion by maansch** [forntoh/LcdMenu#116] [Q&A] [answered]
Hi Thomas,
If the #include instruction <LcdMenu.h> is not in the last position, ITEM_TOGGLE will not work.

If menu.toggleBacklight() is used there is an error
>> Compilation error: 'class LcdMenu' has no member named 'toggleBacklight'
Otherwise it runs through and only the text is displayed in the menu.

Example:  Callbacks.ino  v3.3.0

#include <LcdMenu.h>
#include <ItemSubMenu.h>
#include <ItemToggle.h>

It does not work!

#include <ItemSubMenu.h>
#include <ItemToggle.h>
#include <LcdMenu.h>

is working .

Maybe you should point that out.

br
Marc-Andre

**Comment by forntoh** [comment, marked as answer]
https://lcdmenu.forntoh.dev/usage#1.-include-the-lcdmenu-library-in-your-sketch
